API_doc/Communication.md

```diff
@@ -12,7 +12,7 @@ Each response contain at least **REQUEST_STATE** (see below).
 |*CONFIG_LOAD*| Load an existing config. |**CONFIG_KEY** *or* **READONLY_CONFIG_KEY**| **CONFIG_NAME**<br>**CONFIG_ID**|
 |*CONFIG_UNLOAD*| Unload config. |**CONFIG_ID**|*none*|
 |*CONFIG_INCLUDE*| Include a config in another |**CONFIG_ID**<br>**SRC**<br>**DEST**|*none*|
-|*SETTING_UPDATE*| Update or create setting |**CONFIG_ID**<br>**SETTING_NAME**<br>**SETTING_VALUE**|*none*|
+|*SETTING_UPDATE*| Update or create setting |**CONFIG_ID**<br>**SETTINGS_TO_UPDATE**|*none*|
```

Comment:
Unclear what this means in the context of the file. Please add details above / below.